### PR TITLE
Split ocp install

### PR DESCRIPTION
**Pull request by ebattat**
I have split install steps to:
1. Update OCP version
2. Install using remote ssh direct through GitHub actions shell
3. Verify that install complete successfully.

### CODE REVIEW

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 # This is a nightly CI Pipeline against Performance environment using IPI installer - run on Sunday
 name: Deploy IPI Perf Env Nightly CI
-#on:
-#  schedule:
-#    - cron: '0 4 * * 0' # run on Sunday at 4 AM UTC/ 0 AM EST
 on:
-  push:
-    branches: [ main ]
+  schedule:
+    - cron: '0 4 * * 0' # run on Sunday at 4 AM UTC/ 0 AM EST
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Perf_Env_IPI_CI.yml:7]: Nit: 4 AM UTC is 11 PM EST or midnight EDT.  Just note that it's 4 AM UTC.

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml [hunk 2]
```diff
@@ -73,12 +73,20 @@ jobs:
...
+          podman run --rm -e OCP_ENV_FLAVOR="PERF" -e IBM_API_KEY="$IBM_API_KEY" -e GITHUB_TOKEN="$GITHUB_TOKEN" -e INSTALL_OCP_VERSION="latest-4.8" -e INSTALL_STEP="update_ocp_version" -e OCP_VERSION_BUILD="ga" -e PERF_WORKER_IDS="$PERF_WORKER_IDS" -e PERF_PROVISION_IP="$PERF_PROVISION_IP" -e PERF_CONTAINER_PRIVATE_KEY_PATH="$PERF_CONTAINER_PRIVATE_KEY_PATH" -e CONTAINER_PRIVATE_KEY_PATH=$CONTAINER_PRIVATE_KEY_PATH -e PERF_PROVISION_USER="$PERF_PROVISION_USER" -e PERF_PROVISION_OC_USER="$PERF_PROVISION_OC_USER" -e PERF_PROVISION_PORT="$PERF_PROVISION_PORT" -e PERF_PROVISION_KUBEADMIN_PASSWORD_PATH="$PERF_PROVISION_KUBEADMIN_PASSWORD_PATH" -e PERF_PROVISION_KUBECONFIG_PATH="$PERF_PROVISION_KUBECONFIG_PATH" -e PERF_PROVISION_INSTALLER_PATH="$PERF_PROVISION_INSTALLER_PATH" -e PERF_PROVISION_INSTALLER_CMD="$PERF_PROVISION_INSTALLER_CMD" -e PERF_PROVISION_INSTALLER_LOG="$PERF_PROVISION_INSTALLER_LOG" -e GITHUB_REPOSITORY_SHORT=$GITHUB_REPOSITORY_SHORT -e PERF_PROVISION_TIMEOUT="3600" -e log_level="INFO" -v "$PERF_PROVISION_PRIVATE_KEY_PATH":"$CONTAINER_PRIVATE_KEY_PATH" --privileged "quay.io/ebattat/benchmark-runner:v$build_version" </dev/null
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Perf_Env_IPI_CI.yml:78]: CONTAINER_PRIVATE_KEY_PATH=$CONTAINER_PRIVATE_KEY_PATH: quote "$CONTAINER_PRIVATE_KEY_PATH"

Likewise GITHUB_REPOSITORY_SHORT=$GITHUB_REPOSITORY_SHORT

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml [hunk 3]
```diff
@@ -73,12 +73,20 @@ jobs:
...
+          #ssh -o ServerAliveInterval=30 -o ServerAliveCountMax=5 -o StrictHostKeyChecking=no -t -i $PERF_PROVISION_PRIVATE_KEY_PATH $PERF_PROVISION_USER@$PERF_PROVISION_IP "ibmcloud login --apikey $IBM_API_KEY 1>/dev/null 2>&1 ; $PERF_PROVISION_INSTALLER_CMD"
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Perf_Env_IPI_CI.yml:83]: I know it's commented out, but fix quotes (PERF_PROVISION_PRIVATE_KEY_PATH)

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml [hunk 4]
```diff
@@ -73,12 +73,20 @@ jobs:
...
+          podman run --rm -e OCP_ENV_FLAVOR="PERF" -e IBM_API_KEY="$IBM_API_KEY" -e GITHUB_TOKEN="$GITHUB_TOKEN" -e INSTALL_OCP_VERSION="latest-4.8" -e INSTALL_STEP="verify_install_complete" -e OCP_VERSION_BUILD="ga" -e PERF_WORKER_IDS="$PERF_WORKER_IDS" -e PERF_PROVISION_IP="$PERF_PROVISION_IP" -e PERF_CONTAINER_PRIVATE_KEY_PATH="$PERF_CONTAINER_PRIVATE_KEY_PATH" -e CONTAINER_PRIVATE_KEY_PATH=$CONTAINER_PRIVATE_KEY_PATH -e PERF_PROVISION_USER="$PERF_PROVISION_USER" -e PERF_PROVISION_OC_USER="$PERF_PROVISION_OC_USER" -e PERF_PROVISION_PORT="$PERF_PROVISION_PORT" -e PERF_PROVISION_KUBEADMIN_PASSWORD_PATH="$PERF_PROVISION_KUBEADMIN_PASSWORD_PATH" -e PERF_PROVISION_KUBECONFIG_PATH="$PERF_PROVISION_KUBECONFIG_PATH" -e PERF_PROVISION_INSTALLER_PATH="$PERF_PROVISION_INSTALLER_PATH" -e PERF_PROVISION_INSTALLER_CMD="$PERF_PROVISION_INSTALLER_CMD" -e PERF_PROVISION_INSTALLER_LOG="$PERF_PROVISION_INSTALLER_LOG" -e GITHUB_REPOSITORY_SHORT=$GITHUB_REPOSITORY_SHORT -e PERF_PROVISION_TIMEOUT="3600" -e log_level="INFO" -v "$PERF_PROVISION_PRIVATE_KEY_PATH":"$CONTAINER_PRIVATE_KEY_PATH" --privileged "quay.io/ebattat/benchmark-runner:v$build_version" </dev/null
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Perf_Env_IPI_CI.yml:87]: Same quoting as line 78.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml (hunk 1 above)
Comment (ebattat) [reply .github/workflows/Nightly_Perf_Env_IPI_CI.yml:7]: midnight EDT.

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml (hunk 3 above)
Comment (ebattat) [reply .github/workflows/Nightly_Perf_Env_IPI_CI.yml:83]: ok

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml (hunk 4 above)
Comment (ebattat) [reply .github/workflows/Nightly_Perf_Env_IPI_CI.yml:87]: Done

File: .github/workflows/Nightly_Perf_Env_IPI_CI.yml (hunk 1 above)
Comment (RobertKrawitz) [reply .github/workflows/Nightly_Perf_Env_IPI_CI.yml:7]: US/Eastern *Standard* Time is 5 hours behind UTC.  Eastern *Daylight* Time is 4 hours behind UTC:

```
$ date
Mon 08 Nov 2021 10:24:57 AM EST
$ date -u
Mon 08 Nov 2021 03:24:59 PM UTC
```

Comment (ebattat) [reply .github/workflows/Nightly_Perf_Env_IPI_CI.yml:7]: OK I will change the comment like this in all the nightly CI:
    - cron: '0 4 * * *' # run at 4 AM UTC/ 0 AM EDT

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**